runway/data_types.py

```diff
@@ -405,27 +407,37 @@ class file(object):
         import runway
         from runway.data_types import file, category
 
-        inputs = {"folder": file(is_folder=True)}
+        inputs = {"directory": file(is_directory=True)}
         outputs = {"result": category(choices=["success", "failure"])}
         @runway.command("batch_process", inputs=inputs, outputs=outputs)
         def batch_process(result_of_setup, args):
-            result = do_something_with(args["folder"])
+            result = do_something_with(args["directory"])
             return { "result": "success" if result else "failure" }
 
     :param name: The name of this variable, defaults to None
     :type name: string, optional
-    :param is_folder: Does this variable represent a folder instead of a file? Defaults to False.
-    :type is_folder: bool, optional
+    :param is_directory: Does this variable represent a directory instead of a file? Defaults to False.
+    :type is_directory: bool, optional
     """
 
-    def __init__(self, name=None, is_folder=False):
+    def __init__(self, name=None, is_directory=False, suffix=None):
```

Comment:
What do you think of the name `extension` or `file_extension` over `suffix`? That vocabulary seems to be a bit more common when speaking of file types. I think either is fine, I just bring it up because some people might me more familiar with the term `extension`.

Reply:
Makes sense. `extension` is better!